web/src/auth/ForgetPage.js

```diff
@@ -65,7 +65,10 @@ class ForgetPage extends React.Component {
     ApplicationBackend.getApplication("admin", this.state.applicationName)
       .then((res) => {
         if (res.status === "error") {
-          Setting.showMessage("error", res.msg);
+          this.onUpdateApplication(null);
```

Comment:
why set null in here? and why we need msg to render large message, we haven't this demand in the issue

Reply:
This is for UI design, which allows the front-end to directly display the error message in the entire window after receiving an IP ban error. This implementation method refers to the implementation method of loginPage.js and SignupPage.js.

Reply:
according to 

https://github.com/casdoor/casdoor/blob/347b25676f8a28d06c0a576d1ba1f8ebef80aa6f/web/src/auth/ForgetPage.js#L151-L154

the core api of forgePassword is `SetPassword` in 

https://github.com/casdoor/casdoor/blob/347b25676f8a28d06c0a576d1ba1f8ebef80aa6f/controllers/user.go#L458

Reply:
I didn't add the ip check in setPassword, I will add that later.

Reply:
> This is for UI design, which allows the front-end to directly display the error message in the entire window after receiving an IP ban error. This implementation method refers to the implementation method of loginPage.js.

so why not use Setting.showMessage("error", res.msg);